cross/xcb/Makefile

```diff
@@ -0,0 +1,27 @@
+PKG_NAME = libxcb
```

Comment:
Usually we try to keep the folder name and the PKG_NAME the same. Please follow this rule/guideline for constancy.